packages/vue/src/components/molecules/SfRange/SfRange.stories.js

```diff
@@ -0,0 +1,109 @@
+import { storiesOf } from "@storybook/vue";
+import {
+  withKnobs,
+  number,
+  text,
+  boolean,
+  optionsKnob as options,
+} from "@storybook/addon-knobs";
+import SfRange from "./SfRange.vue";
+storiesOf("Molecules|Range", module)
+  .addDecorator(withKnobs)
+  .add("Common", () => ({
+    props: {      
+      min: {
+        default: number("min", 0, "Props"),
+      },
+      max: {
+        default: number("max", 10, "Props"),
+      },
+      valueMin: {
+        default: number("Value min", 0, "Props"),
+      },
+      valueMax: {
+        default: number("Value Max", 1, "Props"),
+      },
+      step: {
+        default: number("step", 1, "Props"),
+      },
+      label: {
+        default: text("label", "Price", "Props"),
+      },
+      disabled: {
+        default: boolean("disabled", false, "Props"),
+      },
+      orientation: {
+        default: boolean("orientation", true, "Props"),
+      },
+      direction: {
```

Comment:
same here :) I'd change this name on ltrDirection

Reply:
Like above